extensions/omni_python/docs/intro.md

```diff
@@ -0,0 +1,272 @@
+           -e POSTGRES_USER=omnigres \
+           -e POSTGRES_DB=omnigres \
+           --mount source=omnigres,target=/var/lib/postgresql/data -v $(pwd)/python-files:/python-files \
+           -p 127.0.0.1:5433:5432 --rm ghcr.io/omnigres/omnigres-slim:latest
+```
+
+Let's try it out!
+```postgresql
+omnigres=# select hello();
+   hello
+------------
+ Hey there!
+```
+```postgresql
+omnigres=# select add(5, 10);
+ add
+-----
+  15
+```
+```postgresql
+omnigres=# select subtract(5, 10);
+ subtract
+----------
+       -5
+```
+
+## Flask
+
+For Flask framework integration, we have a few more steps.
+
+Create `omni_httpd` extension to be able to handle HTTP requests.
+```postgresql
+create extension if not exists omni_httpd cascade;
+```
+
+Add the following in `requirements.txt` file.
+```
+omni_http[Flask]
+```
+
+Create HTTP listeners for our Flask application. For example, let's add a HTTP listener for port 5000.
+```postgresql
+with
+    listener as (insert into omni_httpd.listeners (address, port) values ('0.0.0.0', 5000) returning id),
+    handler as (insert into omni_httpd.handlers (query) values
+                                                 (
+                                                 $$ select handle(request.*) from request$$
+       ) returning id)
+insert
+into
+    omni_httpd.listeners_handlers (listener_id, handler_id)
+select
+    listener.id,
+    handler.id
+from
+    listener,
+    handler;
+```
+
+Let's say you have a table called `employees`.
+```postgresql
+create table employees (
+   id uuid,
+   name text not null,
+   department text not null,
+   salary integer not null
+);
+```
+
+Now you can update your Python files (in the mounted volume) to include Flask functionality. For example, you can define endpoints to fetch list of all employees, fetch a particular employee, as well as create a new employee record.
+``` py hl_lines="26-30"
+from omni_python import pg
+from omni_http import omni_httpd
+from omni_http.omni_httpd import flask
+from flask import Flask, jsonify, make_response
+import uuid
+
+app = Flask('myapp')
+
+def result_to_json(result):
+    rows = []
+    for row in result:
+        rows.append(dict(row))
+    return json.dumps(rows)
+
+@app.route('/employees', methods=['POST'])
+def create_employee():
+    from flask import make_response, request
+    json_data = json.loads(request.data.decode('UTF-8'))
+
+    employee_id = uuid.uuid4()
+    employee_name = json_data.get('name')
+    employee_department = json_data.get('department')
+    employee_salary = json_data.get('salary')
+
+    if not employee_name or not employee_department or not employee_salary:
+        return "Missing required fields", 400
+
+    employee = plpy.execute(plpy.prepare("INSERT INTO employees (id, name, department, salary) VALUES ($1, $2, $3, $4) RETURNING *", ["uuid", "text", "text", "int"]), [employee_id, employee_name, employee_department, employee_salary])
+    return result_to_json(employee)
```

Comment:
Nitpicking: I haven't checked my code, but it may be more elegant/Pythonesque to write something like 

```python
return json.dumps([dict(employee) for employee in employees])
```

Whether here or in `result_to_json` (`employees_to_json`?)